app/views/nonprofits/show.html.erb

```diff
@@ -6,6 +6,24 @@
   <% end %>
 </h1>
 
+<% unless @nonprofit.social_profile.nil? %>
+  <% if @nonprofit.social_profile.facebook.present? %>
```

Comment:
This was really bothering me. I tried to write a method in the `User` model to make this action more clear. I believe the basic idea was something like this:

```
def has_profile(account)
  social_profile.account.present?
end
```

Then, for each of these I could pass the following:
`<% if @nonprofit.has_profile(facebook) %>`

However, I kept getting an error saying that the local variable 'facebook' did not exist. What am I missing?